tests/drivers/adc/adc_api/src/test_adc.c

```diff
@@ -51,6 +51,7 @@
 	defined(CONFIG_BOARD_ARDUINO_NANO_33_BLE) || \
 	defined(CONFIG_BOARD_BMD_345_EVAL) || \
 	defined(CONFIG_BOARD_UBX_BMD300EVAL_NRF52832) || \
+	defined(CONFIG_BOARD_UBX_BMD330EVAL_NRF52810) || \
```

Comment:
You could list this above nRF52832, to preserve alphabetical order

Reply:
I was following our main board name order (BMD-300, BMD-330, BMD-340, BMD-360), rather than the MCU name.
I'd like to re-do the BMD-345 to follow the same naming structure as well, then slip it in between BMD-340 and BMD-360. I'm not sure of the best way to do that.